### ISSUE TITLE
[Spree Upgrade] Replace order.shipping_method_id in basic_enterprise_serializer

### ISSUE THREAD

**Issue by luisramos0** [openfoodfoundation/openfoodnetwork#2730]
app/serializers/api/admin/basic_enterprise_serializer.rb:
    1  class Api::Admin::BasicEnterpriseSerializer < ActiveModel::Serializer
    2:   attributes ... :shipping_method_ids

**Comment by luisramos0** [comment]
This serializer is used to inject data into the dom in:

- these 2 views:
   - app/views/admin/enterprise_relationships/_data.html.haml
   - app/views/admin/enterprise_roles/_data.html.haml
- as both:
   -  "ofn.admin"."my_enterprises"
   - and "ofn.admin"."all_enterprises"

And also used to inject data into this view: 
- app/overrides/spree/admin/payment_methods/_form/replace_form_fields.html.haml.deface
- as "admin.paymentMethods"."shops"

**Comment by luisramos0** [comment]
There's no order.shipping_method_id in this serializer, it's enterprise.shipping_method_ids.

All right, I have just learned some active record magic [here](https://guides.rubyonrails.org/v4.2/association_basics.html#has-many-association-reference):
"When you declare a has_many association, the declaring class automatically gains 16 methods related to the association"
One fo then is `_ids` so
In Enterprise
`has_many :shipping_methods`
creates `shipping_method_ids`
that is used in the basic_enterprise_serializer.

So, because distributors_shipping_methods is still available in 2-0-stable, nothing will change here and there's nothing to do here.
Closing issue.
